installer/roles/local_docker/tasks/compose.yml

```diff
@@ -18,22 +18,22 @@
 
 - name: Create Docker Compose Configuration
   template:
-    src: "{{ item }}.j2"
-    dest: "{{ docker_compose_dir }}/{{ item }}"
-    mode: 0600
-  with_items:
-    - environment.sh
-    - credentials.py
-    - docker-compose.yml
-    - nginx.conf
-    - redis.conf
+    src: "{{ item.file }}.j2"
+    dest: "{{ docker_compose_dir }}/{{ item.file }}"
+    mode: "{{ item.mode }}"
+  loop:
+    - file: environment.sh
+      mode: "0600"
+    - file: credentials.py
+      mode: "0600"
+    - file: docker-compose.yml
+      mode: "0600"
+    - file: nginx.conf
+      mode: "0600"
+    - file: redis.conf
+      mode: "0666"
```

Comment:
this shouldn't be world writable. 664 or 660.

Reply:
I've updated my PR and changed the file mode to 0664. This is also working.
As I understood the problem here is, that the redis process inside the awx_redis container is running as user redis. But the bind mount file is owned by root:root. So we need at least read rights for anyone.
A nicer solution could be the usage of docker volumes instead of this bind mount solution.